scripts/run_e2e_tests.sh

```diff
@@ -106,7 +106,7 @@ function run_test {
   while true
   do
     # Run next test
-    streamlit "$@" &
+    streamlit "$@" --global.logLevel="error" &
```

Comment:
Why set log level to error here?

Reply:
It looks like the default log level is debug (I believe because development mode is true) making the console very unhelpful for E2E results.

Here's a sample output from before:
https://circleci.com/gh/streamlit/streamlit/6912

Reply:
We can see the individualized log output for each test on the [`Tests` tab](https://app.circleci.com/pipelines/github/streamlit/streamlit/2965/workflows/5c2a3ec5-4c87-4cae-952a-15bcb60124be/jobs/6897) in CircleCI.

In the case where we have a failing test, it's useful to have `debug` logging